dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -218,15 +219,14 @@ mydec_sales AS (
             COUNT() OVER (
                 PARTITION BY line_1_primary_pin, line_4_instrument_date
             ) AS num_single_day_sales,
-            year_of_sale
+            year_of_sale AS year
         FROM {{ source('sale', 'mydec') }}
-        WHERE line_2_total_parcels = 1 -- Remove multisales
     )
     /* Some sales in mydec have multiple rows for one pin on a given sale date.
     Sometimes they have different dates than iasworld prior to 2021 and when
     joined back onto unique_sales will create duplicates by pin/sale date. */
     WHERE num_single_day_sales = 1
-        OR (YEAR(mydec_date) > 2020)
+        OR YEAR(DATE_PARSE(year, '%Y')) > 2020
```

Comment:
```suggestion
        OR year > 2020
```

Will this cause an issue? Might have to wrap 202o in single quotes.